### PR TITLE
Branch p

### PR DESCRIPTION
**Pull request by Rushollow**
perfect request

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 
 print("Hi from K_branch")
 # Sanya prochitai i postav' like
+
+
+def foo():
+    pass
```
Comment (manof356) [inline main.py:7]: add return 1+1

File: main.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
 
 print("Hi from K_branch")
 # Sanya prochitai i postav' like
+
+
```
Comment (Rushollow) [inline main.py:5]: Add more empty lines

File: main.py (hunk 1 above)
Comment (manof356) [reply main.py:7]: pogodi

Comment (Rushollow) [reply main.py:7]: no fck u

**Pull request merged**